contrib/swtpm/template.py

```diff
@@ -0,0 +1,61 @@
+pkgname = "swtpm"
+pkgver = "0.8.1"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--with-gnutls"]
+configure_env = {"NOCONFIGURE": "1"}
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "bash",
+    "expect",
+    "gawk",
+    "gmake",
+    "gnutls-progs",
+    "libtool",
+    "pkgconf",
+    "socat",
+]
+makedepends = [
+    "gnutls-devel",
+    "json-glib-devel",
+    "libseccomp-devel",
+    "libtasn1-devel",
+    "libtpms-devel",
+    "linux-headers",
+    "openssl-devel",
+]
+checkdepends = [
+    "iproute2",
+    "trousers",
+]
+depends = ["gnutls-progs"]
+pkgdesc = "TPM emulator with socket, character device, and Linux CUSE interface"
+maintainer = "cesorious <cesorious@gmail.com>"
+license = "BSD-3-Clause"
+url = "https://github.com/stefanberger/swtpm"
+source = (
+    f"https://github.com/stefanberger/swtpm/archive/refs/tags/v{pkgver}.tar.gz"
+)
+sha256 = "7bba52aa41090f75087034fac5fe8daed10c3e7e7234df7c9558849318927f41"
+
+
+def post_install(self):
+    self.install_license("LICENSE")
+
+
+@subpackage("libswtpm")
+def _libs(self):
+    return ["usr/lib/swtpm/*.so.*"]
+
+
+@subpackage("libswtpm-devel")
+def _devel(self):
+    return self.default_devel()
+
+
+@subpackage("libswtpm-devel-static")
+def _devel(self):
+    self.depends = [f"libswtpm-devel~{pkgver}"]
+
+    return ["usr/lib/swtpm/libswtpm_libtpms.a"]
```

Comment:
```suggestion
```

don't need this, default split does this from devel already

Reply:
without this, cbuild fails with
 `=> swtpm-0.8.1-r0: static libraries should be in the -static package`

Reply:
sure, that seems to happen because of the subdir

Reply:
so just leave it as is? or pull it into devel?

Reply:
i think you can put it into devel yeah

Reply:
or maybe not, if not then it's fine as is

Reply:
cbuild still complains if it's in devel, leaving it as is